src/components/dialogs/parameters/load-flow-parameters.js

```diff
@@ -328,12 +334,17 @@ const SpecificLoadFlowParameters = ({
         return extractDefaultMap(specificParamsDescription);
     }, [specificParamsDescription]);
 
+    const lfParamsRef = useRef();
+    lfParamsRef.current =
+        lfParams?.specificParametersPerProvider?.[currentProvider];
+
+    // When provider changes or defaultValues then we must reset currentParameters state
     useEffect(() => {
         setCurrentParameters({
             ...defaultValues,
-            ...lfParams?.specificParametersPerProvider?.[currentProvider],
+            ...lfParamsRef.current,
         });
-    }, [lfParams, currentProvider, defaultValues]);
+    }, [currentProvider, defaultValues]);
```

Comment:
it looks like we dont need currentProvider as dep anymore.
eslint does not say anything, curious.

Reply:
done